lib/tessel/deploy.js

```diff
@@ -275,7 +275,7 @@ actions.sendBundle = function(tessel, filepath, opts) {
             }
 
             // Log write
-            logs.info('Writing %s to %s on %s (%d kB)...', project.entryPoint, memtype, tessel.name, bundle.length / 1000);
+            logs.info('Writing project to %s on %s (%d kB)...', memtype, tessel.name, bundle.length / 1000);
```

Comment:
Nice! Was going to make an issue for this because the `build.js` was confusing after deploying `index.js`.